relay-cogs/src/lib.rs

```diff
@@ -141,6 +141,8 @@ pub enum AppFeature {
     MetricsSessions,
     /// Metrics in the custom namespace.
     MetricsCustom,
+    /// Metrics in the `metric_stats` namespace.
+    MetricsMetricStats,
```

Comment:
if we rename `MetricStats` -> `Stats` than this will be just clear and without duplication.

Reply:
It looks weird, but imo it makes sense, `Metrics` is the cogs classifier and `MetricStats` is the namespace.